docs/examples/linux_kernel.md

```diff
@@ -0,0 +1,99 @@
+# Comparing Linux kernels
+
+The main focus of DiffKemp is on Red Hat Enterprise Linux (RHEL), whose kernel
+includes a list of functions (kABI symbols) whose semantics should remain the
+same across the lifetime of a single major release. In this example, we will
+show how to compare two versions of the kernel. If you want to try this on your
+own, be aware that it can take a few hours to get the results.
```

Comment:
Hours, really? :D Doesn't the comparison take like 5 minutes?

Reply:
I removed the sentence. The sentence was not very well put, I meant the whole example with everything can take even a few hours -- downloading and preparing the kernels with `rhel-kernel-get` (2x ~50s--1.5min), running `diffkemp build` (2x ~35--50min), running `diffkemp compare`.